introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<p>Is there an opportunity to compare HPO tuned models and regularly trained models here in terms of prediction outputs? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3623/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>That's a good point, and it would help customers quantify the impact of using HPO vs Training, but this will require creating a separate endpoint for the model trained with Training (not a problem), and in general will increase the scope of the notebook. We could instead decide to have a dedicated "HPO vs Training" notebook whose purpose is to show that better performance could be achieved with a tuned model. I'd refrain though from adding this comparison in all the notebooks that I'm modifying (50+). What do you think? </p>

Reply:
<p>What you said makes sense. Maybe after we've created this versus notebook we can come back and add a reference to it in a comment or something. </p>